### PR TITLE
README update.

### PR DESCRIPTION
**Pull request by aslepakurov**
## Description of changes in release / Impact of release:
README update according to latest changes

### Is this a breaking change?
  - [ ] Yes
  - [X] No

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
(insert text here)

### CODE REVIEW

**Review by vvfesik — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -10,12 +10,21 @@
...
+    * [Configurations](#configurations)
+* [How to use](#how-to-use)
+    * [How to generate route](#how-to-generate-route)
+    * [How to manage routes](#how-to-manage-routes)
 * [Reverse proxy mode](#reverse-proxy-mode)
 <!-- tocstop -->
 
+## Prerequisites
+
+- python >3.8.0 (python --version)
```
Comment (rozenbom) [inline README.md:25]: =>3.8.0

**Review by rozenbom — APPROVED** [review]

**Comment by aslepakurov** [comment]
merge after #58

File: README.md [hunk 2]
```diff
@@ -81,17 +160,13 @@ _Note: this manual of how to use vgs-satellite assuming you are running from doc
 
    ![secure-payload](manual/3-secure-payload.png)
    
-1. Check field you would like to secure.
+1. Check field you would like to reveal, choose `Reveal` in `Opearion` dropdown.
```
Comment (vvfesik) [inline README.md:163]:
```suggestion
1. Check field you would like to reveal, choose `Reveal` in `Operation` dropdown.
```

**Review by vvfesik — APPROVED** [review]

**Review by vvfesik — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -104,25 +179,17 @@ _Note: this manual of how to use vgs-satellite assuming you are running from doc
...
-   
-## Reverse proxy mode
+## Mitmproxy
 
-VGS Satellite can be run in reverse proxy mode. For this add `mode: reverse:upstream.local` to the end of ~/.mitmproxy/config.yaml.
-You would need to run request directly to `localhost:9099` without -x(proxy option) in #3 and #13 
\ No newline at end of file
+VGS Satellite's core depends on [mitmproxy](https://github.com/mitmproxy/mitmproxy/). 
+**mitmproxy** or man-in-the-middle proxy is an interactive intercepting proxy with ton of build-in functionalities and protocol support.
+VGS Satellite is provided as a Open Source product under Apache License v2.0
```
Comment (rozenbom) [inline README.md]: Is it really necessary to mention Mitmporxy?

**Pull request merged**